plugins/automox/help.md

```diff
@@ -1188,50 +815,25 @@ Example input:
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|event|event|True|Event with details|
+|devices|[]device|False|List of Automox managed devices|
 
 Example output:
 
 ```
-{
-  "event": {
-    "id": 1234,
-    "name": "system.delete",
-    "server_id": 1234,
-    "organization_id": 1234,
-    "data": {
-      "ip": "192.168.0.1",
-      "os": "Ubuntu",
-      "systemname": "hostname-1"
-    },
-    "server_name": "hostname-1",
-    "create_time": "2021-12-16 06:20:38.153713"
-  }
-}
 ```
 
-### Custom Output Types
-
-#### device
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|Device ID|integer|True|The device ID|
-|Device Name|string|True|The device name|
-|Organization ID|integer|True|The organization ID of the device|
-|Server Group ID|integer|True|The server group ID of the device|
-|Device UUID|string|True|The device unique identifier|
-
-
 ## Troubleshooting
 
 _This plugin does not contain any troubleshooting information._
 
 # Version History
 
-* 1.2.0 - Get device by IP and Get device by hostname: fix validation issue when IP or hostname not found | Add unit tests 
+* 2.0.0 - Fix Vulnerability Sync API Actions
```

Comment:
Is it possible to list what actions have been updated and that the SDK is also updated here please?